front/app/containers/Authentication/Modal.tsx

```diff
@@ -129,7 +132,10 @@ const AuthModal = ({ setModalOpen }: Props) => {
   });
   const fullscreenModalEnabled = _fullscreenModalEnabled && false;
 
-  const closable = currentStep !== 'closed' && currentStep !== 'success';
+  const closable =
+    currentStep !== 'closed' &&
+    currentStep !== 'success' &&
+    currentStep !== 'clave-unica:email';
```

Comment:
Just wondering why the modal isn't closable during the clave-unica step?

Reply:
I just wanted to reduce the number of cases that need to be handled, and I assumed it would be easier with not closable modal. Also, it doesn't make sense for me to register a user and let them use the platform without an email.